CHANGELOG.md

```diff
@@ -585,65 +585,70 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [0.1.0.16] - 2018-11-21
 
+### Added
+
+- _WD_ExecuteScript: Add async support
+
 ### Changed
 
-- Add async support to _WD_ExecuteScript
-- Add debug info to _WD_GetMouseElement
+- _WD_GetMouseElement: Add debug info
 
 ### Fixed
 
-- Set element value in _WD_ElementAction
-- Prevent premature exit in _WD_WaitElement
+- _WD_ElementAction: Set element value
+- _WD_WaitElement: Prevent premature exit
 - ChromeDriver now uses goog:chromeOptions
 
 ## [0.1.0.15] - 2018-09-15
 
 ### Added
 
 - _WD_LoadWait
+- _WD_Option: Add support for BinaryFormat option
 
 ### Changed
 
 - Add support for Unicode text to _WD_ElementAction's "value" option
```

Comment:
proposal
`_WD_ElementAction: - added support for Unicode text to "value" option`